.github/workflows/request_codeowners_review.yml

```diff
@@ -8,11 +8,16 @@ on:
 jobs:
   auto_request_review:
     runs-on: ubuntu-latest
+    permissions: write-all
     steps:
+      - name: Calculate fetch-depth
+        id: calc
+        run: echo "::set-output name=depth::$(echo $((${{ github.event.pull_request.commits }} + 1)))"
```

Comment:
::set-output is [deprecated](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/)

Reply:
Will fix now, thanks